environment.yml

```diff
@@ -2,33 +2,33 @@
 # mamba env create -f environment.yml
 name: myria3d
 channels:
+  - pytorch
+  - nvidia
+  - pyg
+  - comet_ml
   - conda-forge
   - anaconda
   - pytorch
   - comet_ml
   - pyg
 dependencies:
-  - python==3.9.*
+  - python=3.9.*
   - pip
   # --------- data formats --------- #
-  - numpy
   - h5py
+  - numpy
   # --------- Deep Learning --------- #
-  # cudatoolkit to specify the cuda driver in the conda env
-  - conda-forge::cudatoolkit=11.3.1 # single equal sign there, not a typo
-  - pytorch::pytorch==1.11.0
-  - pytorch::torchvision==0.12.0
-  - conda-forge::pytorch-lightning==1.5.9
-  - conda-forge::torchmetrics==0.7.*
-  - comet_ml::comet_ml==3.31.*
-  - conda-forge::urllib3<2  # To solve for https://github.com/GeneralMills/pytrends/issues/591
-  - pyg::pytorch-cluster
-  - pyg::pytorch-scatter
-  - pyg::pytorch-sparse
-  - pyg::pyg==2.1.0
+  # --------- Deep Learning --------- #
+  - pytorch=2.1
```

Comment:
why removing the channel information? I think I added it to make sure to use the right channel for each library. But maybe there is no confusion with the newer versions